src/Pages/Chart/Chart.js

```diff
@@ -1,7 +1,279 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import TotalTable from "./Components/TotalTable";
+import ChartGenre from "./Components/ChartGenre";
+import axios from "axios";
+import * as config from "../../config";
+import ChartHead from "./Components/ChartHead";
+import Modal from "./Components/Modal";
+import styled from "styled-components";
 
 const Chart = () => {
-  return <div>hi</div>;
+  // 데이터받아오기
+  const [totalData, setTotalData] = useState([]);
+
+  useEffect(() => {
+    getTotalData();
+  }, [setTotalData]);
+
+  const getTotalData = async () => {
+    let result = await axios.get("data/total.json");
+    setTotalData(result.data.total);
+  };
+
+  const [value, setValue] = useState("");
+  //국내 해외
+  const handleChangeBtn = e => {
+    setValue(e.target.value);
+    switch (value) {
+      case "total":
+        const getTotalData = async () => {
+          const result = await axios.get(`data/${value}.json`);
+          return setTotalData(result.data.total);
+        };
+        getTotalData();
+        break;
+      case "domestic":
+        const getDomesticData = async () => {
+          const result = await axios.get(`data/${value}.json`);
+          return setTotalData(result.data.domestic);
+        };
+        getDomesticData();
+        break;
+      case "foreign":
+        const getForeignData = async () => {
+          const result = await axios.get(`data/${value}.json`);
+          return setTotalData(result.data.foreign);
+        };
+        getForeignData();
+        break;
+      default:
+    }
+  };
```

Comment:
```suggestion
  const handleChangeBtn = async e => {
    const { value } = e.target;
    const result = await axios.get(`data/${value}.json`);

    setValue(value);
    setTotalData(result.data[value]);
  };
```
- getTotalData 함수를 새로 정의하고 호출하기 보다는 handleChangeBtn 자체에 async 적용
- bracket notation을 활용하여 인자로 받은 값을 setTotalData 함수의 인자로 적용
- 코드 적용해보시고 확인 바랍니다.

Reply:
와.....이렇게 간단하게 할 수 있다니...감사합니다!